.github/scripts/validate_framer_security.py

```
@@ -0,0 +1,251 @@
+
+        # Scan code for NEXT_PUBLIC_ usage
+        code_vars = set()
+        for file_path in Path('src').rglob('*.{js,jsx,ts,tsx}'):
+            try:
+                with open(file_path, 'r', encoding='utf-8') as f:
+                    content = f.read()
+                    # Find process.env.NEXT_PUBLIC_*
+                    matches = re.findall(r'process\.env\.(NEXT_PUBLIC_\w+)', content)
+                    code_vars.update(matches)
+            except Exception:
+                continue
```

Ruff (0.14.10)

`try`-`except`-`continue` detected, consider logging the exception

(S112)